designdocs/extended_token_lock.md

```diff
@@ -0,0 +1,162 @@
+# 📄 Boosting Rewards Extension
+
+## 📚 Context and Scope
+
+This design introduces a new boosting option in the Frequency runtime. It defines a mechanism for locking tokens that is both time-sensitive and event-driven, with immutable thawing behavior.
+
+While the core `Provider Boosting` system remains unchanged, this proposal builds upon it to incentivize long-term token commitments. Users who participate in extended locks will receive differentiated rewards via minimal extensions to the existing `capacity` pallet.
+
+## ❗ Problem Statement
+
+Business requirements call for a mechanism that allows users to:
+
+- Lock tokens before a **Precipitating Tokenomic Event (PTE)**.
```

Comment:
Precipitating > Participating

Reply:
@JoeCap08055 Thoughts about this? I could actually see either one working.